src/config/dynamic-theme-fixes.config

```diff
@@ -7147,6 +7147,13 @@ CSS
 
 ================================
 
+mail.tutanota.com
+
+INVERT
```

Comment:
```suggestion
IGNORE INLINE STYLE
```

Reply:
Ignore inline style doesn't work as this specific selector by itself doesn't appear to have inline style. The QR code remains invisible with this.
Another solution could be to hard code the fill value for this selector. Hard code because the qr should be black on white regardless of the theme. The white part already exists with dark reader enabled.

`CSS
div.mb > div > svg > path {
    fill: black !important;
}`

Contributor guidelines say to not hard code color codes but wrapping it as ${black} makes dark reader make it white when dark mode is enabled.
 
Please tell me if I should try something else.

Reply:
> Ignore inline style doesn't work as this specific selector by itself doesn't appear to have inline style. The QR code remains invisible with this.

You sure? You might wanna change the selector to `div.mb > div > svg *` as INVERT is a possible solution, but not something that should be used in this situation.